react-nativescript-navigation/src/native-stack/views/FrameNavigatorView.tsx

```diff
@@ -5,8 +5,8 @@
 import * as React from 'react';
 // import { View, StyleSheet, Platform } from 'react-native';
 import {
-  ScreenStack,
-  Screen as ScreenComponent,
+  NativeScreensProvider as ScreenStack,
+  NativeScreen as ScreenComponent,
```

Comment:
Just a note to myself to look at the consequences of changing ScreenStack from strictly `"frame"` to `NativeScreensProvider`. I'm nervous about regressing other features to support shared transitions.